Geometry/HGCalGeometry/src/HGCalGeometry.cc

```diff
@@ -278,7 +274,7 @@ HGCalGeometry::CornersVec HGCalGeometry::getCorners(const DetId& detid) const {
   unsigned int cellIndex = indexFor(detid);
   HGCalTopology::DecodedDetId id = m_topology.decode(detid);
   if (cellIndex < m_cellVec2.size() && m_det == DetId::HGCalHSc) {
-    GlobalPoint v = getPosition(detid);
+    GlobalPoint v = getPosition(detid, false);
```

Comment:
Given the default for the second parameter, explicitly adding it here was not strictly necessary. It doesn't hurt, anyhow